### PR TITLE
Con 157 mising vpc subnet

### PR DESCRIPTION
**Pull request by x2adrew**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -2,6 +2,13 @@ variables:
     VPC_NAME:
         description: the vpc to launch in
         required: true
+    VPC_CIDRS:
```
Comment (pauldallen) [inline config.yaml]: CIDRS should be CIDR - also, do we want CIDR or OCTET?

File: services/config.rb [hunk 2]
```diff
@@ -4,12 +4,19 @@
...
-##   action :sustain
-##   cidr "12.0.0.0/16"
-##   internet_gateway true
-## end
-##
+
+coreo_aws_vpc_vpc "${VPC_NAME}" do
+   action :find
+   cidr "${VPC_CIDRS}"
+   tags [
```
Comment (pauldallen) [inline services/config.rb]: This forces a search on a VPC name - i dont like that... perhaps just a blank "tags" var

File: services/config.rb [hunk 3]
```diff
@@ -4,12 +4,19 @@
...
+          "Name=${VPC_NAME}"
+        ]
+end
+
+coreo_aws_vpc_subnet "${PRIVATE_SUBNET_NAME}" do
```
Comment (pauldallen) [inline services/config.rb]: this finds all subnets, you need a route table to define it as private

File: config.yaml (hunk 1 above)
Comment (x2adrew) [inline config.yaml]: Changed to CIDR just to be consistent with leader-elect-cluster, but I don't mind changing to OCTET if that's the standard we want to use.

**Comment by x2adrew** [comment]
Please review this again when you have a chance

**Pull request closed without merging**